### ISSUE TITLE
Static content in tab order - failure of 2.4.3 or not?

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag3#174]
A discussion that seems to keep bubbling up (most recently on WebAIM https://webaim.org/discussion/mail_thread?thread=10037) is whether or not static content that receives focus (e.g. because it has been marked with `tabindex="0"`) is a failure of 2.4.3 or not.

My take for the longest time has been that, unless the *order* that these static elements receive focus in makes stuff illogical/affects meaning of the overall content, that this is fine. I don't think static content receiving focus violates the normative wording of the SC

> If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability.

I also don't see anything in the understanding document that suggests anything about static content being focusable as being a failure.

To be clear, I would generally flag inappropriately focusable static content as something that a developer should address, but I make the point of clarifying that it doesn't strictly (to me at least) fail 2.4.3 (or any other SC). That it's more of a usability problem.

Long story short: it would be good to get a consensus position on this, and to add an example/clarification/note to the understanding document about this.

**Comment by patrickhlauke** [comment]
@kfranqueiro @mbgower it seems this issue was transferred in January to the `wcag3` repo, but it's still an issue that relates to WCAG 2.x. suggest it gets either transferred back ... and if it was added to 3 purely because of "this is something we'll want to tackle in wcag3", then a new separate fresh issue is opened in `wcag3` about it
